scripts/test-resurrect.sh

```
@@ -0,0 +1,120 @@
+WS1="/tmp/herdr-resurrect-ws1-$$"
+WS2="/tmp/herdr-resurrect-ws2-$$"
+LOG="/tmp/herdr-resurrect-test-$$.log"
+
+cleanup() {
+    local code=$?
+    echo "=== Cleanup ===" >&2
+    pkill -f "$TEST_SESSION" 2>/dev/null || true
+    rm -rf "$WS1" "$WS2" 2>/dev/null || true
+    rm -f "$LOG" 2>/dev/null || true
+    exit $code
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)